File: submissions/sm_030_shivansh/week_03/day_3/session_2/count_vowels.js
// functions to count numbers of vowels in a string

function countVowels(str) {
  var count = 0;
  for (var i = 0; i < str.length; i++) {
    switch (str[i]) {
      case 'a':
      case 'A':
      case 'e':
      case 'E':
      case 'i':
      case 'I':
      case 'o':
      case 'O':
      case 'u':
      case 'U':
        count++;
      default:
        continue;
    }
  }
  return count;
}

console.log(countVowels('Siddharth'));

// Input - "shivansh"  "SHIVANSH"  "Siddharth"

File: submissions/sm_026_rohit-kumar/week_14/day_3/evaluation/count_vowels.py
def count_vowels(string):
    lower = 0
    upper = 0

    for ch in string:
        if ch in ['a', 'e', 'i', 'o', 'u']:
            lower = lower + 1
            continue
        
        if ch in ['A', 'E', 'I', 'O', 'U']:
            upper = upper + 1
    
    print('lower - ', lower)
    print('upper - ', upper)


string = input('Enter string : ')
count_vowels(string)

